src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,139 @@
+package duke;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with interactions with the user.
+ */
+public class Ui {
+
+    private Scanner scanner;
+    private final String line = "--------------------------------------------------------------------------------\n";
+
+    public Ui() {
+        scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Prints welcome message.
+     */
+    public String printWelcomeMessage() {
+        String str = "";
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        str += "Welcome to Duke bot!\n";
+        str += "What tasks do you have to do today?\n";
+        str += "To input a deadline or event, type the date and time in the format 'yyyy-mm-ddThh:mm'";
+        return str;
+    }
+
+    /**
+     * Prints goodbye message.
+     */
+    public String printGoodbyeMessage() {
+        return "Bye! See you soon!";
+    }
+
+    /**
+     * Prints list of tasks.
+     */
+    //public String printList(ArrayList<Task> tasks) {
```

Comment:
I noticed that there are quite a lot of commented out code in some files. Would it be neater to delete them if you do not plan on using them again?